gorums.proto

```diff
@@ -12,6 +12,7 @@ extend google.protobuf.MethodOptions {
   bool multicast = 50003;
   bool correctable = 50004;
   bool unicast = 50005;
+  bool rpc = 50006;
```

Comment:
We can remove `ordered`. Also update numbering so it is consecutive. And maybe we should make the `per_node_arg` number `50100` to give more room for call type options.